src/orb.yml

```diff
@@ -204,33 +204,45 @@ executors:
       version:
         type: string
         default: '8'
+      resource_class:
+        type: string
+        default: medium
```

Comment:
is medium the lowest? Perhaps we should add a comment noting that this needs to be enabled by CircleCI w/ a link to the support page to do so?

Reply:
Added a link to the first occurrence of the resource_class parameter.
https://github.com/echobind/circleci-react-native-orb/blob/6996c2f31b959a63d33054fd537dfb1909c4b1ab/src/orb.yml#L208-L209